iroha/src/query.rs

```diff
@@ -190,9 +190,11 @@ mod tests {
         let asset_definition_id = AssetDefinitionId::new("rose", "wonderland");
         let asset_id = AssetId::new(asset_definition_id, account_id);
         let mut store = Metadata::new();
-        let _ = store.insert(
+        let _ = store.insert_with_limits(
             "Bytes".to_string(),
             Value::Vec(vec![Value::U32(1), Value::U32(2), Value::U32(3)]),
+            100,
```

Comment:
Would compilation here fail? If no should we also introduce limits like before here?

Reply:
Yes, I fixed this, pushed just now.